core/editor/src/document/document_message_handler.rs

```diff
@@ -566,33 +574,25 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 					responses.push_back(operation.into());
 				}
 			}
-			ReorderSelectedLayers(delta) => {
-				let selected_layer_paths: Vec<Vec<LayerId>> = self.selected_layers_sorted();
+			MoveSelectedLayersTo { path, insert_index } => {
+				responses.push_back(DocumentMessage::CopySelectedLayers.into());
```

Comment:
Will this copy pollute the user's copy-paste history? Will they be able to undo this operation as a single atomic action?

Reply:
Yes, this would pollute the history if we were to use the same clipboard, but we could think about adding different clipboards because that will come in handy anyway at some point

Reply:
This will be addressed in https://github.com/GraphiteEditor/Graphite/issues/298